assets/scss/app.scss

```diff
@@ -28,6 +28,29 @@
   width: 100%;
 }
 
+header {
+  margin-top: 1.25em;
+  margin-bottom: 0.5em;
+}
+
+ul {
+  @extend .text;
+  @extend .list;
+  @extend .list-bullet;
+}
+
+a {
+  @extend .govuk-link;
+}
+
+header {
+  margin-top: 0;
+}
```

Comment:
Did you meant to have the `header` twice and the margin-top is different on both